### ISSUE TITLE
Displaying results

### ISSUE THREAD

**Issue by LKedward** [fortran-lang/benchmarks#3]
I think it would be good if the final output is HTML+JSON that we can serve up at `fortran-lang.org/benchmarks` via gh-pages.
This can include various customizable and interactive plots to allow easily comparing different scenarios, and links for raw CSV and JSON download.

Some additional features that would be really nice for each benchmark case are the ability to show compiler reports for optimisation and vectorisation as well as disassembled output like godbolt.org for checking generated instructions.

**Comment by certik** [comment]
I agree. (Down the road this effort can lead to having hundreds of benchmarks and compiler comparison on these.)
